doc/api/index.rst

```diff
@@ -0,0 +1,20 @@
+API Documentation
+=================
+
+.. toctree::
```

Comment:
I think it's better if the module docstring comes first, and afterwards the sub-pages.

Reply:
Done.